lib/chef/knife/bootstrap/chef_vault_handler.rb

```diff
@@ -141,7 +142,7 @@ def load_chef_bootstrap_vault_item(vault, item)
         def require_chef_vault!
           @require_chef_vault ||=
             begin
-              error_message = "Knife bootstrap requires version 2.6.0 or higher of the chef-vault gem to configure chef vault items"
+              error_message = "Knife bootstrap requires version 2.6.0 or higher of the chef-vault gem to configure vault items"
```

Comment:
Geez we should just remove this entirely. That version shipped 4 years ago

Reply:
You _know_ someone out there is still running an older version, though.

Reply:
Well tell @robbkidd to upgrade